build.gradle

```diff
@@ -44,6 +44,7 @@ subprojects {
             exceptionFormat = 'full'
             events "STARTED", "PASSED", "FAILED", "SKIPPED"
         }
+        failFast = true
```

Comment:
committed by mistake, or?
Will make it hard to understand the CI status for the contributors

Reply:
Yes, will remove